resources/i18n/zh_CN/uranium.po

```diff
@@ -0,0 +1,749 @@
+# Uranium
+# Copyright (C) 2017 Ultimaker
+# This file is distributed under the same license as the Uranium package.
+# Ruben Dulek <r.dulek@ultimaker.com>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Uranium 2.6\n"
+"Report-Msgid-Bugs-To: r.dulek@ultimaker.com\n"
+"POT-Creation-Date: 2017-05-30 15:41+0200\n"
+"PO-Revision-Date: 2017-06-27 20:18+0800\n"
+"Last-Translator: PCDotFan <pc@edu.ax>\n"
+"Language-Team: PCDotFan <pc@edu.ax>\n"
+"Lang-Code: xx\n"
+"Country-Code: XX\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"X-Generator: Poedit 1.7.3\n"
+"Language: zh_CN\n"
```

Comment:
Could you change the Language field to be a human-readable string such as "Simplified Chinese"? Thanks.